.github/workflows/test.yml

```diff
@@ -26,5 +26,6 @@ jobs:
       - name: Run flake8
         run: flake8 app/
       - name: Run tests
+        timeout-minutes: 5
```

Comment:
The `timeout-minutes` setting is used to specify the maximum time a step can run before being automatically canceled. Ensure that 5 minutes is sufficient for your test suite to complete. If your tests typically take longer, consider increasing this value to avoid premature termination.